<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toFile</class>
 <widget class="QDialog" name="toFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Save sheet to file</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/tofile</normaloff>:/new/prefix1/tofile</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Format</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>jpg</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>png</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>bmp</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quality</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>341</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Y =</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Automatically set the height</string>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Transparent background (only png)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
